File: 백준/Silver/2839. 설탕 배달/설탕 배달.py
#2839
import sys

def input():
    return sys.stdin.readline().rstrip()

def solve():
    result = 0
    n = int(input())
    
    while n >= 0:
        if n % 5 == 0:
            result += (n // 5)
            print(result)
            return
        n -= 3
        result += 1
        
    print(-1)
    
solve() 

File: 백준/Silver/2839. 설탕 배달/설탕 배달.cc
//2839
#include <iostream>

using namespace std;

int main() {
	int n, result = 0;
	cin >> n;

	while (n >= 0) {
		if (n % 5 == 0) {
			result += (n / 5);
			cout << result << '\n';

			return 0;
		}
		n -= 3;
		result++;
	}
	cout << -1 << '\n';
}